terraform/variables.tf

```diff
@@ -45,6 +45,7 @@ variable key_vault_infra_secret_name {}
 
 variable azure_credentials { default = null }
 
+variable "restore_db_from_instance" { default = null }
```

Comment:
both DBs will have to be in the same plan for this work. and restore from snapshot option only works for small plans and above.